src/Microsoft.Diagnostics.TestHelpers/AcquireDotNetTestStep.cs

```diff
@@ -168,7 +168,12 @@ private static async Task Unzip(string zipPath, string expandedDirPath, ITestOut
                 ZipArchive zip = new(zipStream);
                 foreach (ZipArchiveEntry entry in zip.Entries)
                 {
-                    string extractedFilePath = Path.Combine(expandedDirPath, entry.FullName);
+                    string extractedFilePath = Path.GetFullPath(Path.Combine(expandedDirPath, entry.FullName));
+                    string fullExtractedDirPath = Path.GetFullPath(expandedDirPath + Path.DirectorySeparatorChar);
```

Comment:
I am not sure this is enough:

`C:\foo\bar` and entry `..\baz` -> this test passes.

Reply:
Yes, GetFullPath will convert relative paths into absolute ones.